### ISSUE TITLE
Admin Panel

### ISSUE THREAD

**Issue by Anmolbansal1** [scorelab/fact-Bounty#394]
An admin panel is required for application which allows admin to interact with private routes like cron job routes and system meta information.

**Comment by hv7214** [comment]
@Anmolbansal1  is this being worked upon?

**Comment by carpecodeum** [comment]
@hv7214 yes i am currently working on it just waiting for a review of the maintainers
